assets/openai.yaml

```diff
@@ -932,6 +932,32 @@
   prohibited_uses: ''
   monitoring: ''
   feedback: ''
+- type: model
+  name: GPT-4 Turbo
```

Comment:
update the downstream dependencies (I assume GPT-4 Turbo is available via OpenAI API; is it available via ChatGPT?)